src/bindings/python/tests/test_inference_engine/test_function.py

```diff
@@ -271,3 +271,39 @@ def test_evaluate_invalid_input_shape():
             [Tensor("float32", Shape([3, 1])), Tensor("float32", Shape([3, 1]))],
         )
     assert "must be compatible with the partial shape: {2,1}" in str(e.value)
+
+
+def test_get_batch():
+    param1 = ops.parameter(Shape([2, 1]), dtype=np.float32, name="data1")
+    param2 = ops.parameter(Shape([2, 1]), dtype=np.float32, name="data2")
+    add = ops.add(param1, param2)
+    func = Function(add, [param1, param2], "TestFunction")
+    param = func.get_parameters()[0]
+    param.set_layout(Layout("NCHW"))
+    assert get_batch(func) == 2
+
+
+def test_get_batch_CHWN():
+    param1 = ops.parameter(Shape([3, 1, 3, 3]), dtype=np.float32, name="data1")
+    param2 = ops.parameter(Shape([3, 1, 3, 3]), dtype=np.float32, name="data2")
+    param3 = ops.parameter(Shape([3, 1, 3, 3]), dtype=np.float32, name="data3")
+    add = ops.add(param1, param2)
+    add2 = ops.add(add, param3)
+    func = Function(add2, [param1, param2, param3], "TestFunction")
+    param = func.get_parameters()[0]
+    param.set_layout(Layout("CHWN"))
+    assert get_batch(func) == 3
```

Comment:
can we set it to '4'? So that it will be different with first dimension='3'?

Reply:
done